gufe/setup/network_planning/network_plan.py

```diff
@@ -0,0 +1,191 @@
+# This code is part of gufe and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/gufe
+from __future__ import annotations
+
+from itertools import chain
+import json
+import networkx as nx
+from typing import FrozenSet, Iterable, Optional
+import gufe
+
+from gufe import Component
+from gufe.tokenization import GufeTokenizable
+
+from .component_mapping import ComponentMapping
+
+class NetworkPlan(GufeTokenizable):
```

Comment:
Very briefly looking at class names, I don't think `Plan` is the right word here. It's not immediately clear what it does.

Reply:
do you have an alternative suggestion?

Reply:
@dotsdl  maybe we can have a short look at this this evening in the gufe meetin?

Reply:
This Class is required for this PR, I'm open for name suggestions, however a `NetworkPlan` is the result of a `NetworkPlanner` ;).